src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsList.tsx

```diff
@@ -77,7 +80,7 @@ const ConditionsList: React.FC<IConditionsListProps> = (props) => {
                 valueRangeType,
                 getNextColor(valueRanges, defaultSwatchColors)
             ),
-            selectedTarget: `#${LIST_KEY}`
+            selectedTarget: `#${CONDITIONS_ADD_BUTTON_ID}`
```

Comment:
any reason we don't do the `useId` thing? I don't know why that is beneficial but we do it most places.

Reply:
yeah we use useId for target stuff which ensures uniqueness of the generated string